meta-resin-common/recipes-support/resinhup/resinhup/run-resinhup.sh

```diff
@@ -265,8 +265,8 @@ if [ "$LOG" == "yes" ]; then
 fi
 
 # Get the slug
-if [ -f /mnt/conf/config.json ]; then
```

Comment:
Very good catch @michal-mazurek . But we should keep them both so old device can run find the config too. Basically we need another elif.